test/files/run/pure-warning-post-macro/test_2.scala

```diff
@@ -0,0 +1,13 @@
+// scalac: -Xfatal-errors
```

Comment:
should be `-Xfatal-warnings` but it's "fine" as 2.12 doesn't support the pragmas, only `.flags` files 😉

Reply:
https://github.com/scala/scala/pull/9071